chsdi/templates/htmlpopup/gewaesserschutz_badewasserqualitaet.mako

```diff
@@ -76,7 +77,11 @@
   </tr>
     <tr>
     <td class="cell-meta">${_('ch.bafu.gewaesserschutz-badewasserqualitaet.url')}</td>
+% if c['attributes']['url']:
```

Comment:
as a rule of thumb, the datenherr-provided URLs are not checked (see pdfs too). It would still do no harm to do a check